core/src/buffer.rs

```diff
@@ -37,12 +37,45 @@ use scale_info::{
 #[derive(Clone, Default, Eq, Hash, Ord, PartialEq, PartialOrd, Decode, Encode, TypeInfo)]
 pub struct LimitedVec<T, E, const N: usize>(Vec<T>, PhantomData<E>);
 
+impl<T: Clone + Default, E: Default, const N: usize> Display for LimitedVec<T, E, N>
```

Comment:
please state in comments to this impl (can be done above the `impl`) that you use "+" for printing full vector.
The idea is to state semantics of custom precision symbols.